userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
+     "http://127.0.0.1:8080/plugins/killbill-analytics/reports?name=report_accounts_summary&startDate=2018-01-01&endDate=2022-01-01&sqlOnly=true"
+----
+
+This returns the SQL query corresponding to the `report_accounts_summary` report.
+
+==== Refresh a report - to be confirmed
+
+[source,bash]
+----
+curl -v \
+     -X PUT \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     -H 'Content-Type: application/json' \
+     -d '{"reportPrettyName": "New accounts created (refreshed at 2am GMT)",
+          "refreshFrequency": "DAILY",
+          "refreshHourOfDayGmt": 2}' \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/reports/new_accounts_per_day?shouldRefresh=true"	 
+----
+
+== Data API
+
+The analytics plugin provides a Data API that supports some account level operations as explained below.
+
+=== Retrieve Analytics Data for Account
+
+You can retrieve all analytics data for an account as follows:
+
+[source,bash]
+----
+curl -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/<account_id>"
+----
+
+This returns the data from the analytics tables for the specified `account_id`.
+
+=== Refresh an Account
+
+You can force a refresh for a given account as follows:
+
+[source,bash]
+----
+curl -v \
+     -X PUT \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/<account_id>"
+----
+
+This refreshes the analytics tables for the specified `account id`.
+
+=== Refresh all Accounts
+
+You can refresh all accounts as follows:
+
+[source,bash]
+----
+curl -s \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     "http://127.0.0.1:8080/1.0/kb/accounts/pagination" | \
+ruby -r json -e 'JSON.parse(gets).map { |a| puts a["accountId"] }' | \
+xargs -I accountId \
+curl -v \
+     -X PUT \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+    "http://127.0.0.1:8080/plugins/killbill-analytics/accountId"
+----
+
+[[analytics_tables]]
+== Analytics Tables
+
+This section gives a brief overview of some of the important tables in the analytics database.
+
+=== Accounts
+
+The table *analytics_accounts* maintains information about Kill Bill accounts. The table doesn't contain historic records (it will contain at most one record per Kill Bill account).
+
+=== Subscriptions
+
+The table *analytics_subscription_transitions* maintains information about Kill Bill subscriptions. It describes the history of changes (both past and future) on a per subscription basis. It contains all phase transitions for all subscriptions and can be used to build subscription timelines. A few remarks:
+
+* Given a subscription id, events are sorted chronologically when stored (you can rely on `record_id asc`)
```

Comment:
```suggestion
* Given a subscription ID, events are sorted chronologically when stored (you can rely on `record_id asc`)
```

Reply:
Shoudl "record_id asc" be "record_id_asc"?

Reply:
No, `record_id asc` is correct.